src/verso.rs

```diff
@@ -50,6 +50,7 @@ pub struct Verso {
     windows: HashMap<WindowId, (Window, DocumentId)>,
     compositor: Option<IOCompositor>,
     constellation_sender: Sender<ConstellationMsg>,
+    ipc_sender: Option<IpcSender<VersoMessage>>,
```

Comment:
I think the sender might need a better naming.

`ipc_sender` sounds ambiguous. If we want to make the naming consistent, it could be `verso_sender` that sending `VersoMessage`.

`ConstellationMsg` are messages that are handled by constellation.
`EmbedderMsg` are messages that are handled by embedded.
So the `VersoMessage` should be messages that are handled by Verso?

Reply:
> ipc_sender sounds ambiguous. If we want to make the naming consistent, it could be verso_sender that sending VersoMessage.

Yeah, it's indeed confusing, maybe `to_controller_sender`? Adding in a `to` makes it easy to understand where it's gonna go

> ConstellationMsg are messages that are handled by constellation.
EmbedderMsg are messages that are handled by embedded.
So the VersoMessage should be messages that are handled by Verso?

To be honest, I would probably want to name them `ControllerToVersoMessage` and `VersoToControllerMessage`, or maybe `ToVersoMessage` and `ToControllerMessage`, what do you think?

Reply:
Renamed